SPECS/sos/sos.spec

```diff
@@ -1,25 +1,28 @@
 %{!?python3_sitelib: %global python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
 Summary:        A set of tools to gather troubleshooting information from a system
 Name:           sos
-Version:        4.4
-Release:        2%{?dist}
-License:        GPLv2+
+Version:        4.6.1
+Release:        1%{?dist}
+License:        GPL-2.0-or-later
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/sosreport/sos
 #Source0:       https://github.com/sosreport/sos/archive/%%{version}.tar.gz
```

Comment:
can you uncomment the (real) source line so standard tooling like `spectool -g` will work?

Reply:
i think you also need to update the cgmanifest to get the PR check to stop complaining

Reply:
Okay, I actually updated the spec file to be `https://github.com/sosreport/sos/archive/%{version}.tar.gz`

Reply:
Modified [source0](https://github.com/microsoft/CBL-Mariner/pull/7279/files#diff-13142143e58a92ac0b9072352431d38592b137813459326c03806506653d2894R10) of the spec file instead